src/engine/engine.go

```diff
@@ -410,8 +421,10 @@ func (e *Engine) PrintExtraPrompt(promptType ExtraPromptType) string {
 			return prompt
 		}
 		return str
-	case shell.PWSH, shell.PWSH5, shell.CMD, shell.BASH, shell.FISH, shell.NU:
+	case shell.PWSH, shell.PWSH5, shell.CMD, shell.BASH, shell.FISH, shell.NU, shell.PLAIN:
```

Comment:
Same question as above

Reply:
Same response as above :)
However if there is something I missed in my approach, which results in this being useless, of course feel free to ignore it !